src/gui/libcloudproviders/libcloudproviders.cmake

```
@@ -27,11 +29,11 @@ if(WITH_LIBCLOUDPROVIDERS)
             PkgConfig::GIO
     )
     target_compile_definitions(owncloudCore PRIVATE WITH_LIBCLOUDPROVIDERS)
-    
+
     configure_file(libcloudproviders/cloud-provider.ini.in ${CMAKE_CURRENT_BINARY_DIR}/${APPLICATION_CLOUDPROVIDERS_DBUS_NAME}.ini)
     install(
         FILES ${CMAKE_CURRENT_BINARY_DIR}/${APPLICATION_CLOUDPROVIDERS_DBUS_NAME}.ini
-        DESTINATION "${DATADIR}/cloud-providers")
+        DESTINATION "${CMAKE_INSTALL_DATADIR}/cloud-providers")
```

```suggestion
        DESTINATION "${DATA_INSTALL_DIR}/cloud-providers")
```

should work without `GNUInstallDirs`?